src/styles/_supported-devices.scss

```diff
@@ -0,0 +1,105 @@
+@use './colors';
+
+$row-padding: 8px;
+
+@mixin all {
+  .seam-supported-devices-table-state-block {
+    width: 100%;
+    background-color: colors.$bg-a;
+    padding: 32px 8px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  .seam-supported-devices-table {
+    width: 100%;
+    overflow-x: auto;
+    table-layout: auto;
+    border-collapse: collapse;
+
+    th,
+    td {
+      white-space: nowrap;
+      padding: 0 8px;
+    }
+
+    thead {
+      width: 100%;
+      height: 50px;
+      background: colors.$bg-a;
+      overflow: hidden;
+
+      tr {
+        border-radius: 8px;
+      }
+
+      th {
+        vertical-align: middle;
+        text-align: left;
+        font-size: 16px;
+        font-weight: 600;
+      }
+    }
+
+    tbody {
+      tr {
+        height: 60px;
+        border-bottom: 1px solid colors.$bg-b;
+      }
+
+      td {
+        vertical-align: middle;
+      }
+    }
+
+    // stylelint-disable-next-line no-descending-specificity
```

Comment:
If you move this block above where the linter suggests, it throws more errors saying to move it down. I could probably spend 30-60 minutes rewriting the UI and adding classnames just to make the linter happy, or I could just add this line because I know it's fine and it works.